templates/package/content/container.tmpl

```diff
@@ -36,11 +36,13 @@
 				</thead>
 				<tbody>
 					{{range .PackageDescriptor.Metadata.Manifests}}
-					<tr>
-						<td><a href="{{$.PackageDescriptor.PackageWebLink}}/{{PathEscape .Digest}}">{{.Digest}}</a></td>
-						<td>{{.Platform}}</td>
-						<td>{{FileSize .Size}}</td>
-					</tr>
+						{{if ne .Platform "unknown/unknown"}}
+						<tr>
+							<td><a class="ui sha label" href="{{$.PackageDescriptor.PackageWebLink}}/{{PathEscape .Digest}}">{{StringUtils.TrimPrefix .Digest "sha256:" | ShortSha}}</a></td>
```

Comment:
@silverwind 

I would suggest not adding the "label" style. Here it is not for a label, it is a row cell with important content.

See dockerhub:

![image](https://github.com/user-attachments/assets/560bf9e1-152f-40d4-8e13-89c5918bb658)

Reply:
I think keeping `label` can make our website look more coordinated.